js/index.js

```diff
@@ -1,7 +1,100 @@
 // Iteration 1: Names and Input
 
+const hacker1 = 'Karl';
+
+console.log(`The driver's name is ${hacker1}.`);
+
+const hacker2 = 'Sebastian';
+
+console.log(`The navigator's name is ${hacker2}.`);
+
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+} else if (hacker2.length > hacker1.length) {
+  console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+} else {
+  console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`) 
+  }
+
+
+// Iteration 3: Loops
+
+let finalName = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  finalName += hacker1[i].toUpperCase() + ' ';
+}
+
+console.log(finalName);
+
+let reverseNavigator = "";
+
+for (let i = hacker2.length-1; i >= 0; i--) {
+  reverseNavigator += hacker2[i];
+}
+
+console.log(reverseNavigator);
+
+let comparisonArray = [hacker1, hacker2];
+
+comparisonArray.sort();
+
+console.log(comparisonArray);
+
+if (comparisonArray[0] == comparisonArray[1]) {
+  console.log('What?! You both have the same name?');
+} else if(comparisonArray[0] == hacker1) {
+ console.log('The driver\'s name goes first');
+} else {
+   console.log('Yo, the navigator goes first definitely.');
```

Comment:
nice approach! you could also use localeCompare() as well in this case. https://www.w3schools.com/jsref/jsref_localecompare.asp